IFC4x3/Properties/o/OCSFittingType_2gURBVuJP1TgJZ3PaUKlJ7/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="OCSFittingType_2gURBVuJP1TgJZ3PaUKlJ7" Name="OCSFittingType" UniqueId="aa79b2df-e136-4176-a4e3-0d991e52f4c7" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
I am a bit confused on the location of this property on a fastener. Is this okay?

Reply:
Ok. commit : de4bcf9de7f5e2e1d9e24e66e3b338da3b5506eb